src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -54,19 +83,28 @@ export const generateContent: TranslationFn = content => {
     State.AwaitingFinalOrder,
     State.FinalOrderRequested,
     State.FinalOrderComplete,
-  ];
+  ]);
+  const currentState = content.userCase.state ? states.at(content.userCase.state.toString()) : undefined;
```

Comment:
a case will never be stateless so do we need this condition?

Reply:
the code doesn't compile if I don't do that, I get this error
`ERROR in /home/taleb/projects/reform/repos/nfdiv-dev-env/nfdiv-frontend/src/main/steps/applicant1/hub-page/joint/content.ts
[1] 87:33-55
[1] [tsl] ERROR in /home/taleb/projects/reform/repos/nfdiv-dev-env/nfdiv-frontend/src/main/steps/applicant1/hub-page/joint/content.ts(87,34)
[1]       TS2532: Object is possibly 'undefined'.
`

Reply:
what about `content.userCase.state as State`?

Reply:
if not then its fine

Reply:
that works, I'll do that

Reply:
done